aiplatform/snippets/embeddings_preview.go

```diff
@@ -27,38 +26,34 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
-func embedTextsPreview(
-	apiEndpoint, project, model string, texts []string,
-	task string, dimensionality *int) ([][]float32, error) {
+func embedTextsPreview(project, location, model string) ([][]float32, error) {
```

Comment:
See previous comments:
+ Pass texts in as parameter
+ Provide comments explaining parameters
+ Remove return type (error is okay)